components/content-types/Forms/FormsWidget.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Forms from './Forms';
+const dotCMSApi = require('../../../utils/dotcms/dotcmsApi');
+
+const FormsWidget = ({ formId, rendered }) => {
+    const [items, setItems] = useState([]);
+    let variable;
+
+    if (rendered) {
+        variable = rendered.match(/variable='[^']+('*)/g)[0].split("'")[1];
```

Comment:
Can't we get this from the contentlet object?

Reply:
no, this only comes in the `rendered` attr